cls/SourceControl/Git/Utils.cls

```diff
@@ -1473,7 +1510,7 @@ ClassMethod RunGitCommand(command As %String, Output errStream, Output outStream
 ClassMethod RunGitCommandWithInput(command As %String, inFile As %String = "", Output errStream, Output outStream, args...) As %Integer
 {
     // Special case: git --version is used internally even when the settings incorporated here may be invalid/unspecified.
-    if (command '= "--version") {
+    if (command '= "--version") && (command '= "remote") {
```

Comment:
I'm concerned with omitting remote here - this should follow IPM context and therefore depends on the root. Can you explain?

Reply:
@isc-hwojnick  I'm still concerned about this.

Reply:
@isc-tleavitt I'm pretty sure I resolved this as you requested. Not sure why github doesn't show that